api-reference/10 UI Components/dxValidator/8 Validation Rules/EmailRule/EmailRule.md

```diff
@@ -13,5 +13,9 @@ A validation rule that demands that the validated field match the Email pattern.
     href: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Validation/Overview/"
 }
 
+The Email pattern in DevExtreme components is the following:
```

Comment:
Is it possible to turn this line into an instructional statement? For example:

`Use the following email pattern in DevExtreme components:`